README.md

```diff
@@ -77,14 +77,19 @@ In the `./nodezoo-system` folder run,
 fuge build fuge/system.yml
 ```
 
-### 6. Start the infrastructure
+### 6. Insert GitHub token
+For the GitHub service to work you must insert your GitHub personal access token in to the sample.env file located in fuge/env.
+
+__Note:__ If GitHub fails to authenticate you then it may be because Fuge at minimum version 0.6.1 is required.
```

Comment:
Change to, Fuge 0.6.1 or later is required, put it at the top of the file when you install fuge.